Sources/ReadingListKit/PocketAccessLayer.swift

```diff
@@ -0,0 +1,15 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import Foundation
+
+class PocketAccessLayer {
+    let authTokenProvider: () -> String
+    let consumerKey: String
+
+    init(_ authTokenProvider: @escaping () -> String, _ consumerKey: String) {
+        self.authTokenProvider = authTokenProvider
+        self.consumerKey = consumerKey
+    }
+}
```

Comment:
can you clarify (and maybe add some comments) what this type does? Also, does it have any relationship with the existing `MoSoSession` type?

Reply:
The AccessLayer acts as a translation layer between ReadingListModel and the Pocket GraphQL API.
ReadingListModel can request a page number from a user's Reading List, the AccessLayer is responsible for generating the GraphQL request and attaching the appropriate Bearer Token.